src/app/features/stacks-transaction-request/contract-deploy-details/contract-deploy-details.tsx

```diff
@@ -43,22 +44,21 @@ function ContractCodeSection() {
   );
 }
 
-interface TabButtonProps extends BoxProps {
+interface TabButtonProps extends HTMLStyledProps<'button'> {
   isActive: boolean;
 }
 
 function TabButton(props: TabButtonProps) {
   const { isActive, ...rest } = props;
 
   return (
-    <Caption
-      as="button"
+    <styled.button
+      bg={isActive ? 'accent.component-background-hover' : 'transparent'}
```

Comment:
probably doesn't work? https://panda-css.com/docs/guides/dynamic-styling#runtime-values

Reply:
I thought I did check it, but will double check bc doesn't seem like it should.

Reply:
This does toggle correctly... 🤔 

<img width="220" alt="Screen Shot 2023-11-09 at 9 26 31 AM" src="https://github.com/leather-wallet/extension/assets/6493321/1bd57d39-c723-4b65-8e87-59e481f4cca4">

Reply:
Dynamic styling does work when using a token.

Reply:
https://panda-css.com/docs/guides/dynamic-styling#using-token